### PR TITLE
fix #14404: table schema tab ui issues

### LINKED ISSUE #14404: UI /UX issues
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
UI

**Describe the bug**
- [x] Arrow alignment issue in Schema Table.
- [x] Scroll to top issue after description modal opens/closes.
- [x] Fix permission issue while editing description and retention period 

**To Reproduce**

Screenshots or steps to reproduce
1)
Screenshot 2023-12-15 at 6 07 50 PM
2) 

https://github.com/open-metadata/OpenMetadata/assets/58542468/c84d3e19-1567-44b8-abdd-0ba563cee05b

**Expected behavior**
1) Proper alignment
2) Table shouldn't scroll to top when modal opens/closes.

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by harsh-vador**
### Describe your changes:

Fixes #14404 

- I worked on fixing the alignment of the arrow icon in the schema table. 
- Fixed permission issue while editing description and retention period.
- Fixed the modal scroll issue.

Screenshot 2023-12-15 at 6 15 12 PM

https://github.com/open-metadata/OpenMetadata/assets/58542468/ffbf2197-229c-4024-9825-b3372390aff7

#
### Type of change:

- [x] Bug fix

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/Modals/ModalWithMarkdownEditor/ModalWithMarkdownEditor.tsx [hunk 1]
```diff
@@ -18,7 +18,6 @@ import { useTranslation } from 'react-i18next';
 import { showErrorToast } from '../../../utils/ToastUtils';
 import RichTextEditor from '../../common/RichTextEditor/RichTextEditor';
 import Loader from '../../Loader/Loader';
-import './modal-with-markdown-editor.less';
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/Modals/ModalWithMarkdownEditor/ModalWithMarkdownEditor.tsx:21]: Why did we remove this?

File: openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.component.tsx [hunk 2]
```diff
@@ -76,7 +80,17 @@ const SchemaTable = ({
...
+
+  const editDisplayNamePermission = useMemo(
+    () =>
+      checkPermission(
+        Operation.EditDisplayName,
+        ResourceEntity.TABLE,
+        permissions
+      ),
+    [permissions]
+  );
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.component.tsx:93]: Here we should use evaluated permissions, so use `getEntityPermissionByFqn` and get the permission of the table entity by fqn.

Comment (harsh-vador) [reply openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.component.tsx]: Done

File: openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.component.tsx [hunk 3]
```diff
@@ -79,21 +85,40 @@ const SchemaTable = ({
 
   const [searchedColumns, setSearchedColumns] = useState<Column[]>([]);
   const [expandedRowKeys, setExpandedRowKeys] = useState<string[]>([]);
-
+  const [topicPermissions, setTopicPermissions] =
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.component.tsx:88]:
```suggestion
  const [tablePermissions, setTablePermissions] =
```

Comment (harsh-vador) [reply openmetadata-ui/src/main/resources/ui/src/components/SchemaTable/SchemaTable.component.tsx:88]: done

**Review by Sachin-chaurasiya — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**